src/pybind/rgw/rgw_file.pyx

```diff
@@ -99,7 +99,8 @@ cdef extern from "rados/rgw_file.h" nogil:
         uint64_t  f_files    # inodes
         uint64_t  f_ffree    # free inodes
         uint64_t  f_favail   # free inodes for unprivileged users
-        uint64_t  f_fsid[2]  # /* file system ID
+        uint64_t  f_fsid     # /* file system ID
+        uint64_t  f_fsid2    # /* file system ID 2
```

Comment:
probably, we can return `(statbuf.f_fsid, statbuf.f_fsid2)` in `LibRGWFS. statfs`?